### ISSUE TITLE
Aggregation glorytun UDP doesn't work  for me..

### ISSUE THREAD

**Issue by esquilo444** [Ysurac/openmptcprouter#3363]
Good morning,
I can't get the aggregation to work at all in glorytun UDP, with or without a proxy and whatever the Proxy used. the only one that works in vpn is mlvpn, but it is unstable and therefore unusable for my srt video streams.. With v2ray the UDP aggregation also works, but not stable over time either.. I'm a little desperate. The only solution I see is to use one of the proxies for tcp, they all work quite well and manage to get UDP aggregation working by glorytun UDP which has a very stable rtt...
In short, I am open to any suggestions.
thanks 

Jérôme 

## Specifications

  - OpenMPTCProuter version:  Snapshot 6.6
  - OpenMPTCProuter VPS version: kernel test 6.6
  - OpenMPTCProuter VPS provider:ovh
  - OpenMPTCProuter platform: RPI4

**Comment by Ysurac** [comment]
With Glorytun UDP, you can check status and path using ```glorytun-udp path``` on the router via SSH

**Comment by esquilo444** [comment]
I'm going to try that. I will get back to you...
THANKS
ps udp aggregation via glorytun works for you?

**Comment by esquilo444** [comment]
<img width="512" alt="Capture d'écran 2024-05-17 223633" src="https://github.com/Ysurac/openmptcprouter/assets/88659019/0d68c38b-0708-4cbe-b8ef-97ecbd73d0f4">
<img width="509" alt="Capture d'écran 2024-05-17 231442" src="https://github.com/Ysurac/openmptcprouter/assets/88659019/b37f91a3-f2ef-403f-80df-f549e3904be2">

**Comment by esquilo444** [comment]
i don't no if that can help you ...

**Comment by esquilo444** [comment]
<img width="622" alt="Capture d'écran 2024-05-18 003801" src="https://github.com/Ysurac/openmptcprouter/assets/88659019/4e8abb85-7724-4b14-970b-567ff3e76570">

**Comment by Alison110** [comment]
somethimes I use glorytun tcp at first then if it was ok, I use glorytun udp.
or 
restart your vps, after 2 minutes go to system ->startup and restart glorytun udp.

**Comment by esquilo444** [comment]
I will try it ...
Thanks

**Comment by esquilo444** [comment]
unfortunately this does not work.
maybe it's the SRT protocol I'm using to transport the video that's causing the problem...this protocol has its own error corrector..
Do you have a solution to test aggregation via UDP, without me using my software to transport my videos...
this will start to pinpoint the problem.. thanks for your suggestions.

**Comment by Ysurac** [comment]
To test aggregation, set proxy to none, so all is using the VPN and you can test aggregation.
There is a know issue with SRT, but I'm a real newbie on using SRT, so not easy to test and would need some times...

**Comment by esquilo444** [comment]
Thanks..😅
I am a professional in video and since covid, specialized in streaming, but relatively new to the network part.
I've learned a lot of things in recent years, but I'm failing here, even if I remain convinced that there is a via open mtpcp router...

**Comment by Ysurac** [comment]
There is many draft for thinks over quic, but I didn't found anything already developed that I can use for now.
